.github/PULL_REQUEST_TEMPLATE/release-checklist.md

```diff
@@ -167,6 +167,13 @@ and the updated changelog:
 - [ ] Publish the pre-release to GitHub using "Publish Release"
 - [ ] Delete all the [draft releases from the list of releases](https://github.com/ZcashFoundation/zebra/releases)
 
+## Publish crates
+
+- [ ] Install `cargo-release` if needed: `cargo install cargo-release`
+- [ ] Check that you are logged into `cargo` as an owner of the crate or a member of the `owners` group
+- [ ] Do a dry-run: `cargo release --workspace -v`
+- [ ] Publish the crates in tree-order: `cargo release --workspace -v --execute`
+
 ## Binary Testing
 
```

Comment:
I think we need to test the published crates:
(This can be automated later.)

```suggestion

- [ ] Check that Zebra can be installed from `crates.io`:
        `cargo install --force --version 1.0.0 zebrad && ~/.cargo/bin/zebrad`
```

Reply:
I tested this command after publishing with PR #6908 and it worked.